plugins/pulp_rpm/plugins/importers/yum/importer.py

```diff
@@ -35,23 +35,30 @@ def metadata(cls):
             'types': [
                 ids.TYPE_ID_DISTRO, ids.TYPE_ID_DRPM, ids.TYPE_ID_ERRATA,
                 ids.TYPE_ID_PKG_GROUP, ids.TYPE_ID_PKG_CATEGORY, ids.TYPE_ID_RPM,
-                ids.TYPE_ID_SRPM, models.YumMetadataFile.TYPE, ids.TYPE_ID_PKG_ENVIRONMENT
+                ids.TYPE_ID_SRPM, ids.TYPE_ID_YUM_REPO_METADATA_FILE, ids.TYPE_ID_PKG_ENVIRONMENT
```

Comment:
Unfortunately, when you're accessing the class itself and not an instance, you have to reach deeper to get the value, all the way to ``models.YummetadataFile._content_type_id.default``. This is because the field doesn't have a value until you make an instance and populate it.